packages/hooks/src/errorMessages.ts

```diff
@@ -34,4 +34,6 @@ export enum ErrorMessages {
   OWNERSHIP_NOT_FOUND = 'Ownership not found',
 
   POLLING_TIMEOUT = 'Polling timeout. could not check ownership',
+
+  TRANSACTION_REJECTED_BY_USER = 'You rejected the transaction',
```

Comment:
All error messages are neutral without saying "You"

Reply:
```suggestion
  TRANSACTION_REJECTED_BY_USER = 'Transaction was cancelled',
```